test.js

```diff
@@ -15,3 +15,10 @@ assert.equal(result1, undefined)
 
 const result2 = two.info()
 assert.equal(result2, 'info')
+
+assert.ok(one.child)
+assert.equal(Function.prototype.isPrototypeOf(one.child), true)
+
+const three = two.child()
+assert.ok(three.info)
+assert.equal(Function.prototype.isPrototypeOf(three.info), true)
```

Comment:
Testing for a grandchild by creating a `four` instance from the `three` instance would be a good idea.